hack/create-dummy-cd.py

```diff
@@ -0,0 +1,17 @@
+import os
+import yaml
+
+version = os.environ["VERSION"]
+repo_ctx = os.environ["REPO_CTX"]
+component_descriptor_dir = os.environ["COMPONENT_DESCRIPTOR_DIR"]
+
+cd = None
+
+with open(os.path.join(component_descriptor_dir, "base_component_descriptor_v2"), "r") as base_cd_file:
+    cd = yaml.safe_load(base_cd_file.read())
+
+cd["component"]["version"] = version
+cd["component"]["repositoryContexts"][0]["baseUrl"] = repo_ctx
```

Comment:
Not needed, the pre-generated CD should already have these values set.

Reply:
These values are actually overwritten so that they don't interfere with the component descriptor we push manually.
We are not using this component descriptor. We use it only to satisfy the CICD requirements.
The repository context will replaced with the dev repo and the version will get a "-dummy" postfix.